contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -574,7 +576,7 @@ class MyInputStream : public ANTLRInputStream
     
 		void setText(size_t pos, const char *newText)
 		{
-			UTF32String	newText32 = utf8_to_utf32(newText);
```

Comment:
Why this change happens ?

Reply:
I think that `UTF32String` may have been an acceptable macro in an older C++ version? This was raising a compilation issue after using C++17 for compilation. Based on everything I could find, `std::u32string` seems to be the widely accepted standard type to use for 32-bit character strings.